packages/terra-core-docs/CHANGELOG.md

```diff
@@ -2,29 +2,23 @@
 
```

Comment:
Note: There were some errors/duplicates in the CHANGELOG caused by merge conflicts in a previous commit. This PR corrects those errors and the CHANGELOG should now be accurate.